src/SwedbankPay/Api/Service/Paymentorder/Request/Test.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SwedbankPay\Api\Service\Paymentorder\Request;
+
+use SwedbankPay\Api\Client\Client;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderObject;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderPayeeInfo;
+use SwedbankPay\Api\Service\Paymentorder\Resource\Request\Paymentorder;
+use SwedbankPay\Api\Client\Exception;
+
+class Test
```

Comment:
Is it not possible to create a base class for all of these `Test` classes that does most of the work, catching of the exceptions, etc., as all of that code is duplicated in every `Test` class now?